packages/react-components/src/components/utils.ts

```diff
@@ -410,3 +411,37 @@ export const isEnterKeyEventFromCompositionSession = (e: KeyboardEvent): boolean
  * @private
  */
 export const nullToUndefined = <T>(value: T | null): T | undefined => (value === null ? undefined : value);
+
+/**
+ * @private
+ */
+export const formatMoreButtonAiraDescription = (
+  displayName?: string,
+  isMuted?: boolean,
+  isHandRaised?: boolean,
+  state?: string,
+  isSpeaking?: boolean,
+  strings?: VideoTileStrings
+): string => {
+  let ariaDescription = '';
+  if (displayName) {
+    ariaDescription = ariaDescription += ` ${displayName}`;
```

Comment:
String concatenation with localized strings is bad as languages don't piece together the same. Could we instead do something like:
```ts
/// strings.json
"moreOptionsButtonAriaLabel" : "More Options {participantName} {participantMutedStatus} {participantRaisedHandStatus} {participantIsSpeakingStatus}"
```

Reply:
Not sure if there's a way to mark the templated string as optional? cc @mgamis-msft